examples/express-ts/src/models/Animal.ts

```diff
@@ -9,7 +9,7 @@ export default class Animal extends Model {
   species: string;
 
   // Optional eager relations.
-  owner?: Person;
+  owner?: Partial<Person>;
```

Comment:
Why Partials? That's surprising to me.

Reply:
I'm confused about your confusion. This is our conclusion from #724. It's not strictly needed in this particular example, but not having it demoed anywhere in the examples lead me on quite a goose chase trying to find typings that generally work with eager relations.

Reply:
I was surprised that you used a Partial in the model specification. I expected them to only exist in querybuilder responses.

Reply:
It was your suggestion, and it made things work:
https://github.com/Vincit/objection.js/issues/724#issuecomment-357461867

Reply:
I notice that it's not being tested in `examples.ts`. Were we to test it there, we'd probably make the relation properties all partials in the tests too.